src/MoBi.Presentation/Presenter/ExtendablePathAndValueBuildingBlockPresenter.cs

```diff
@@ -274,5 +265,22 @@ public void Handle(BulkUpdateStartedEvent eventToHandle)
          _focusedStartValue = PathAndValueEntityFrom(_view.FocusedStartValue);
          _handleChangedEvents = false;
       }
+
+      public void AddNewPathAndValueEntity(ObjectPath entityPath) => addNewPathAndValue(entityPath);
+
+      public void AddNewEmptyPathAndValueEntity() => addNewPathAndValue(null);
+
+      private void addNewPathAndValue(ObjectPath entityPath)
```

Comment:
If you make `entityPath` optional then you can call this on line 271 without the null. Just a bit tidier.